src/lib/Mapper/SearchHitToContentSuggestionMapper.php

```diff
@@ -51,6 +53,11 @@ public function map(SearchHit $searchHit): ?ContentSuggestion
             $parentsLocation = array_slice($parentsLocation, (int)$position);
```

Comment:
It's unnecessary to explicitly define the top-level node. Each Root location ought to be set up through configuration. Consider the following:
```suggestion
            $parentsLocation = array_slice($parentsLocation, ++$position);
```
This resolves the problem. Since 'array_slice' accepts an offset as its second parameter, previously I was incorrectly passing an array key of the found element.

Reply:
I think it makes sense to always remove root node from the path. Even if it's narrowed down by Siteaccess configuration, it's still most likely a technical folder and IIRC we don't display it either in the content tree, UDW etc.

Reply:
Unfortunately, for embedding like `/Media`, `/Dashboards`, `/Users` etc is not enough, as they doesn't contains site access root in their location path and mentioned `array_slice` will not be executed at all.